### ISSUE TITLE
Rename the options in particles so they make more sense . .

### ISSUE THREAD

**Issue by jasperbrooks79** [godotengine/godot-proposals#1350]
**Describe the project you are working on:**
A Tomb Raider game, atm . .

**Describe the problem or limitation you are having in your project:**
When I add a particle system, it's not really clear if it's GPU particles, or CPU particles, in the add node . . So, a small request, so for 3D particle systems, they get called GPUParticles, and CPUParticles . . The second thing is, it's very un-intuitive that the two main settings are call ' Process Material ', and ' Draw Passes ' . . Since ' Draw Passes ' is really the mesh, even if it's just a plane, it would make sense if it was called ' Particle Mesh ', and ' Process Material ' was called ' Particle Parameters ' . . There's always a good reason why something is so, in Godot, so I can't really critique it, it would be nice if it was renamed, to make more sense . . Or, just called ' Process Material ' "Parameters", and ' Draw Passes " Particle Mesh 3D " . . :OO . . That way, it's more clear what the settings will actually do, I still keep clicking the wrong place, after using it for over 2 months <3, because of names . . Hope it will be updated, hopeful . . .

**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**
Well, it's just a constant struggle, to have to re-think what one knows, as the current names aren't the best, they're probably very accurate, at a high-tech level, but it's not that easy, for beginners . . I hope you will update the names for Godot, to make more sense, right away . . . Thanks . .

**Describe how your proposal will work, with code, pseudocode, mockups, and/or diagrams:**
Uhmmm, change the names, so they are more user-friendly . .

**If this enhancement will not be used often, can it be worked around with a few lines of script?:**
I'm not good enough, to say if this can be done, as a new user . . 

**Is there a reason why this should be core and not an add-on in the asset library?:**
It would make it easier for all users to use the engine, and make more sense, or so . . .

**Comment by Zireael07** [comment]
+1 to changing header from process material to parameters.

**Comment by TheDuriel** [comment]
See https://github.com/godotengine/godot-docs/issues/2359

**Comment by KoBeWi** [comment]
Related probably #815

**Comment by clayjohn** [comment]
> to changing header from process material to parameters.

While this makes sense for people using a Particles material, it doesn't make sense at all for ShaderMaterials. 

Users will be confused why they are attaching a shader to the properties section of a node.

**Comment by Zireael07** [comment]
@clayjohn This is a suggestion for particles, as far as I understood it (it's a bit rambling)

**Comment by clayjohn** [comment]
@Zireael07 yes, but my comment is regarding changing the "process material" name to "parameters". Process material takes a material, it doesn't take a list of parameters.

**Comment by Zireael07** [comment]
I thought it was a general heading like 'Transform' - my mistake!

**Comment by jasperbrooks79** [comment]
Okay, made a mistake, maybe, always confuses me, the way things are named . .
